### ISSUE TITLE
Suggested aliases

### ISSUE THREAD

**Issue by ewangi** [linuxmint/mintinstall#230]
Can I suggest that the following package names be added to the list of aliases in mintinstall.py:

cairo-dock -> Cairo Dock
chromium-browser -> Chromium
eog - > Eye of GNOME
furiusisomount -> Furius ISO Mount
gimp -> GIMP
gnupg -> GnuPG
gnupg2 -> GnuPG 2
gprename -> GPRename
gthumb -> gThumb
handbrake-gtk -> Handbrake
keepass2 -> KeePass2
keepassx -> KeePassX
mc -> Midnight Commander
openarena -> OpenArena
opera-desktop -> Opera
pcmanfm -> PCManFM
rawtherapee -> RawTherapee
smplayer -> SMPlayer

I can add to this list, but would like to find out if it is deemed worth my while first.

There are various reasons for wanting to add aliases:
(i) Where a package name varies significantly from the commonly known name.
(ii) Where there is an additional part to the package name that is unnecessary because there is only one basic package, e.g. -client, -desktop, -gtk, installer, -launcher, -qt, -stable.
(iii) Where capitalisation is used to define the package name but this is lost if no alias is used, e.g. pcmanfm -> PCManFM.

**Comment by xenopeek** [comment]
Linking to the list of current aliases: https://github.com/linuxmint/mintinstall/blob/master/usr/lib/linuxmint/mintinstall/mintinstall.py#L61. These are used as the display name instead of the package name, also for the purpose of searching.

I think these all make sense because people may likely not know the package name but know the program's name:

> cairo-dock -> Cairo Dock
> chromium-browser -> Chromium
> eog - > Eye of GNOME
> furiusisomount -> Furius ISO Mount
> handbrake-gtk -> Handbrake
> mc -> Midnight Commander
> opera-desktop -> Opera

**Comment by ewangi** [comment]
Thanks for your reply, Vincent.

Given how much I talked about the Opera package in one of the other issues here, yes, the package is _opera-stable_. A big oops there. :)

Re: Chromium, I don't see a package of that name in any of the supported Ubuntu repos (https://packages.ubuntu.com/search?keywords=chromium). As it is, the display name used by mintinstall is only for display - each package listing in mintinstall shows the real package name (although the real package name is *not* shown in the shorter "list" listing - I will open a feature request for that to be included for packages that are using aliases).

So, calling chromium-browser "Chromium" would only potentially confuse those people who had managed to install this other package called chromium from a third-party repo and would only do so in that that they would see two packages called "Chromium" in the list of packages (a confusion which would be resolved by the aforementioned feature request).

Acknowledged re: the title-cased packages. It's all a question of the costs and benefits relating to how easy it would be to put the list together (and maintain it), and the time it would take to generate the aliases, against to what extent this would improve the ease-of-use of mintinstall. You guys would be able to make much more informed decisions on this than I would.

Thanks for the info on KeePassX. As someone who uses the package, interesting to know. I'll move over to either of the suggested packages.

**Comment by xenopeek** [comment]
I could have sworn there was a package called chromium in the Ubuntu repos before, and that's why it was renamed to chromium-browser (as the source package on Debian is called chromium).

As for the title-cased aliases, I don't understand why those are in there so not touching those yet :)

**Comment by ewangi** [comment]
Yup, it's definitely the law of diminishing returns for those aliases which only have capitalisation changes from the package name. I think though that in an ideal world, where there were no issues with list maintenance and processing time, it would be nice to have the display names for all packages (or at least the ones that are commonly used, for some definition of "common") showing the real language name of the package. For example, gimp is "GIMP", "keepassxc" is "KeePassXC", etc. It would be more natural that way for the user.

I'm certainly willing to put in the legwork and produce that list of aliases if you and the rest of the team decide that it is feasible/desirable to implement aliases for packages with simple capitalisation changes.
